readme.md

```diff
@@ -33,6 +33,12 @@ $ de --help
 
 It's `de` and not `del`, since `del` is already a builtin command on Windows.
 
+If preferred, a simple alias could fix this of course.
+
+```
+	alias del=de
```

Comment:
alright! ;)

On Fri, Jan 22, 2016 at 1:07 PM Sindre Sorhus <notifications@github.com>
wrote:

> In readme.md
> <https://github.com/sindresorhus/del-cli/pull/5#discussion_r50530182>:
>
> > @@ -33,6 +33,12 @@ $ de --help
> >
> >  It's `de` and not `del`, since `del` is already a builtin command on Windows.
> >
> > +If preferred, a simple alias could fix this of course.
> > +
> > +```
> > +	alias del=de
>
> @schnittstabil <https://github.com/schnittstabil> I think the current is
> fine. People that don't use the common shells like bash/zsh usually knows
> the difference.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sindresorhus/del-cli/pull/5/files#r50530182>.
>
-- 
--

*Dimitrie Hoekstra, UI & Interaction Designer*
dhesign.com  | dimitriehoekstra@gmail.com | LinkedIn
<https://www.linkedin.com/in/dimitrieh> | Twitter
<https://twitter.com/strages33/>